packages/mobile/src/dappsExplorer/DAppsExplorerScreen.tsx

```diff
@@ -0,0 +1,349 @@
+            </Text>
+
+            {/* <Text>Dont show again Message</Text> */}
+            <Button
+              style={styles.bottomSheetButton}
+              onPress={onNavigationButtonPress}
+              text={t('dappsScreenBottomSheet.button', { dappName: dappSelected?.name })}
+            />
+          </View>
+        </View>
+      </BottomSheet>
+
+      <Dialog
+        title={t('dappsScreenHelpDialog.title')}
+        isVisible={isHelpDialogVisible}
+        secondaryActionText={t('dappsScreenHelpDialog.dismiss')}
+        secondaryActionPress={() => setHelpDialogVisible(false)}
+      >
+        {t('dappsScreenHelpDialog.message')}
+      </Dialog>
+
+      <ScrollView style={styles.scrollContainer}>
+        <View style={styles.descriptionContainer}>
+          <Text style={{ ...fontStyles.h1, flex: 1 }}>{t('dappsScreen.message')}</Text>
+          <View style={styles.descriptionImage}>
+            <DappsExplorerLogo />
+          </View>
+        </View>
+        <>
+          {loading && (
+            <View style={styles.centerContainer}>
+              <ActivityIndicator
+                style={styles.loadingIcon}
+                size="large"
+                color={colors.greenBrand}
+                testID="DAppExplorerScreen/loading"
+              />
+            </View>
+          )}
+          {!loading &&
+            categoriesWithItems.map((category) => renderCategoryWithItems(category, onItemPress))}
+        </>
+      </ScrollView>
+    </SafeAreaView>
+  )
+}
+
+function renderCategoryWithItems(
+  categoryWithItems: CategoryWithItems,
+  onItemPress: (dapp: DappItem) => void
+) {
+  Logger.debug(TAG, `Render category ${JSON.stringify(categoryWithItems)}`)
+
+  return (
+    <View style={styles.categoryContainer} key={`category-${categoryWithItems.id}`}>
+      <View
+        style={{
+          ...styles.categoryTextContainer,
+          backgroundColor: categoryWithItems.backgroundColor,
+        }}
+      >
+        <Text style={{ ...styles.categoryText, color: categoryWithItems.fontColor }}>
+          {categoryWithItems.name}
+        </Text>
+      </View>
+      <>{categoryWithItems.items.map((item) => renderItem(item, onItemPress))}</>
+    </View>
+  )
+}
+
+function renderItem(item: DappItem, onItemPress: (dapp: DappItem) => void) {
+  Logger.debug(TAG, `Render item ${JSON.stringify(item)}`)
+
+  const onPress = () => {
+    return onItemPress(item)
+  }
+
+  return (
+    <Card style={styles.card} rounded={true} shadow={Shadow.Soft} key={`item-${item.id}`}>
+      <TouchableOpacity style={styles.pressableCard} onPress={onPress}>
+        <Image source={{ uri: item.iconUrl }} style={styles.dappIcon} />
+        <View style={styles.itemTextContainer}>
+          <Text style={styles.itemTitleText}>{item.name}</Text>
+          <Text style={styles.itemSubtitleText}>{item.description}</Text>
+        </View>
+        <Image source={require('src/images/link-arrow.png')} style={styles.linkArrow} />
```

Comment:
I think for more icons we add the png in `src/images/Images.ts` and import it wherever it's used instead of requiring them inline like this.

Not sure why we do that, and I realize that there's one more place that does it like this, but I think it makes sense to copy the most common pattern?

Reply:
Yep, you are right, I just copy pasted but it makes sense to continue with the pattern